### PR TITLE
Correct HTMLPortalElement src IDL and content attribute wording.

### LINKED ISSUE #72: Need to separate src IDL and content attributes
Right now there is no definition for what the src IDL attribute does. (It should reflect the src content attribute.)

Relatedly, all references to the src content attribute currently go to the IDL block, which does not define a content attribute.

### LINKED ISSUE #79: "Whenever a portal element’s src attribute is set"
I assume you want this to run on any modifications to the src="" attribute? If so, HTML uses the phrasing

> ... has its src attribute set, changed, or removed, ...

The truly-correct way to do this would be to explicitly set up [attribute change steps](https://dom.spec.whatwg.org/#concept-element-attributes-change-ext), but no existing HTML element does that, so I suspect if we ever rigorize that (https://github.com/whatwg/html/issues/1571) we'll create an easier-to-use abstraction. In the meantime just aping HTML's current style makes sense.

If the intent was truly the first time it was set, then that should be called out explicitly in a note.

### PR DESCRIPTION
**Pull request by jeremyroman**
Resolves #72.
Resolves #79.

***
<a href="https://pr-preview.s3.amazonaws.com/jeremyroman/portals/pull/104.html" title="Last updated on Apr 25, 2019, 3:44 PM UTC (01dc7cf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/portals/104/18b9d1e...jeremyroman:01dc7cf.html" title="Last updated on Apr 25, 2019, 3:44 PM UTC (01dc7cf)">Diff</a>

### CODE REVIEW

**Comment by jeremyroman** [comment]
@domenic

File: index.bs [hunk 1]
```diff
@@ -404,10 +406,10 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
 
     1. [=Assert=]: |element| has a [=guest browsing context=].
 
-    1. If |element| has no {{HTMLPortalElement/src}} attribute specified, or its value is the empty string,
+    1. If |element| has no "`src`" attribute specified, or its value is the empty string,
```
Comment (domenic) [inline index.bs]: You want `<{portal/src}>`. This means you'll need a definition for the `src=""` content attribute (`<dfn element-attr for="portal">src</dfn>`). See e.g. https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-src

This also touches briefly on #71, in that the sentence

> The attribute, if present, must be a valid non-empty URL potentially surrounded by spaces. 

for iframe is an authoring conformance requirement, not an implementer requirement.

Comment (jeremyroman) [reply index.bs]: Man, I Ctrl+F-ed through tabatkins.github.io/bikeshed/ looking for how "element-attr" was spelled and couldn't figure it out. Thanks. :)

Comment (jeremyroman) [reply index.bs]: Done (though I've left #71 as something to do in a later PR, if that's alright).

File: index.bs [hunk 2]
```diff
@@ -240,6 +248,8 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
 
   * <dfn>complete portal activation</dfn>
 
+  The <dfn attribute for="HTMLPortalElement">src</dfn> attribute must [=reflect=] the <{portal/src}> content attribute.
```
Comment (domenic) [inline index.bs]:
```suggestion
  The <dfn attribute for="HTMLPortalElement">src</dfn> IDL attribute must [=reflect=] the <{portal/src}> content attribute.
```

**Pull request merged**